backend/api/Areas/Property/Controllers/SearchController.cs

```diff
@@ -121,6 +121,7 @@ public IActionResult GetGeoJson([FromBody] GeoJsonFilterModel filter)
             if (!filter.IsValid()) throw new BadRequestException("Property filter must contain valid values.");
 
             var pfilter = filter.CopyValues(new AllPropertyFilter());
+            pfilter.PropertyType = filter.PropertyType;
```

Comment:
Tried doing `mapper.Map<AllPropertyFilter>(filter)` but the value still wasn't being sent over. This did allow it to work; however, not sure if it is necessary anymore but thought I would leave it in in case we need to use this filter eventually. Or should I just take it out until we get the mapper/copyvals working?

Reply:
Ah, I'll take a look at this if I find time in a week or so.